diff --git a/reservoir.install b/reservoir.install
index 9427b5c..e7aa17f 100644
--- a/reservoir.install
+++ b/reservoir.install
@@ -45,64 +45,6 @@ function reservoir_install() {
   else {
     drupal_set_message('Could not automatically generate an initial public/private key pair to use for OAuth2 authentication. Until configuring this, OAuth2 authentication will not work.', 'warning');
   }
-
-  // Create demo users & demo client.
-  User::create([])
-    ->setUsername('demo-user')
-    ->setPassword('demo-user')
-    ->activate()
-    ->save();
-  User::create([])
-    ->setUsername('demo-writer')
-    ->setPassword('demo-writer')
-    ->set('roles', ['content_administrator'])
-    ->activate()
-    ->save();
-  User::create([])
-    ->setUsername('demo-developer')
-    ->setPassword('demo-developer')
-    ->set('roles', ['client_developer'])
-    ->activate()
-    ->save();
-  Oauth2Client::create([
-    'owner_id' => 1,
-    'label' => 'Demo app',
-    'secret' => 'foobar',
-    'confidential' => TRUE,
-    'roles' => [],
-  ])->save();
-
-  // Create default content.
-  file_unmanaged_copy(\Drupal::root() . '/' . drupal_get_path('profile', 'reservoir') . '/reservoir-logo.png', 'public://reservoir-logo.png');
-  $file = File::create(['uri' => 'public://reservoir-logo.png']);
-  $file->save();
-  $term = Term::create([
-    'name' => 'Camelids',
-    'vid' => 'tags',
-  ]);
-  $term->save();
-  Node::create([
-    'type' => 'article',
-    'field_body' => [
-      'value' => 'The name "llama" was adopted by European settlers from native Peruvians.',
-      'format' => 'basic_html',
-    ],
-    'field_image' => [
-      [
-        'alt' => 'Reservoir logo: a "pancake stack" with layers of water instead of pancakes, to reference Drupal!',
-        'title' => 'Reservoir logo',
-        'target_id' => $file->id(),
-      ],
-    ],
-    'field_tags' => [
-      [
-        'target_id' => $term->id(),
-      ],
-    ],
-  ])
-    ->setTitle('Hello world!')
-    ->setOwnerId(1)
-    ->save();
 }
 
 // Source: https://stackoverflow.com/a/34962604.
