diff --git a/reservoir.install b/reservoir.install
index 126a943..2608081 100644
--- a/reservoir.install
+++ b/reservoir.install
@@ -73,38 +73,6 @@ function reservoir_install() {
       'client__own_articles',
     ],
   ])->save();
-
-  // Create default content.
-  file_unmanaged_copy(\Drupal::root() . '/' . drupal_get_path('profile', 'reservoir') . '/reservoir-logo.png', 'public://reservoir-logo.png');
-  $file = File::create(['uri' => 'public://reservoir-logo.png']);
-  $file->save();
-  $term = Term::create([
-    'name' => 'Camelids',
-    'vid' => 'tags',
-  ]);
-  $term->save();
-  Node::create([
-    'type' => 'article',
-    'body' => [
-      'value' => 'The name "llama" was adopted by European settlers from native Peruvians.',
-      'format' => 'basic_html',
-    ],
-    'field_image' => [
-      [
-        'alt' => 'Reservoir logo: a "pancake stack" with layers of water instead of pancakes, to reference Drupal!',
-        'title' => 'Reservoir logo',
-        'target_id' => $file->id(),
-      ],
-    ],
-    'field_tags' => [
-      [
-        'target_id' => $term->id(),
-      ],
-    ],
-  ])
-    ->setTitle('Hello world!')
-    ->setOwnerId(1)
-    ->save();
 }
 
 // Source: https://stackoverflow.com/a/34962604.
